.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE_REPOSITORY_CREATION_REQUEST.md

```diff
@@ -0,0 +1,44 @@
+---
+name: "Request New Repository"
+description: "Request the creation of a new repository under DSACMS."
+labels: ["help-desk", "repo-request", "repo-management"]
+assignees: []
+---
+
+## Request a New Repository
+
+If you are looking to migrate an existing repository hosted in a different organization into a new repository under DSACMS, the new repository will be created as private by default. In order for your repository to be released as public, please follow our [outbound review checklist](https://github.com/DSACMS/ospo-guide/blob/main/outbound/outbound-checklists.md) for your project’s maturity tier.
+
+If you are looking to create a completely new repository, your repository can be created as private or public. Based on the maturity model tier selected, a collection of markdown [templates](https://github.com/DSACMS/ospo-guide/blob/main/outbound/repository-templates-and-linters.md) will be included as part of our repository hygiene standards. GitHub security features such as Dependabot and secret scanning will be enabled.
```

Comment:
For a fresh repository, we ask for the request to include an assessment of benefits and risks